### ISSUE TITLE
Editing file in one pane causes same file in other pane to scroll

### ISSUE THREAD

**Issue by chenglou** [atom/atom#10447]
I think ideally this shouldn't be the case? I frequently have the same file open in two panes to view different sections. Editing one and having the other move around defeats the purpose.

**Comment by mnquintana** [comment]
I can't reproduce this in Atom 1.6.0-dev-8677ca3.

- What version of Atom are you using?
- Can you reproduce this after quitting Atom completely and reopening it in safe mode?
    - If you can, can you provide specific step-by-step instructions for reproducing this?

Please take a look at [this guide for submitting bug reports](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#reporting-bugs).

**Comment by chenglou** [comment]
1.4.0. Maybe that's why.

**Comment by mnquintana** [comment]
I can't reproduce this in Atom 1.4.0 either.

- Can you reproduce this after quitting Atom completely and reopening it in safe mode? (`atom --safe`)
    - If you can, can you provide specific step-by-step instructions for reproducing this?

Please take a look at [this guide for submitting bug reports](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#reporting-bugs).

**Comment by chenglou** [comment]
Sorry, maybe scrolling wasn't the correct word? I can reproduce this in safe mode.

1. Open a file, split a pane to the right (two identical files side by side now).
2. Scroll down the one on the right a bit.
3. Go to the beginning of the file on the left, insert a line.
4. See that the one on the right jumped a line. Expected behaviour is that it'd stay still.

![asd-2](https://cloud.githubusercontent.com/assets/1909539/12375073/d5ed5334-bc7f-11e5-923a-bdab18ac78d9.gif)

**Comment by usmanbashir** [comment]
I'm not sure I would call this a bug exactly. But, it's more like a missing feature.

I tested the expected behavior on Atom 1.7.4 and other editors like VS Code 1.1.1 and Vim 7.4. And found that the splits only behave as you @chenglou expected them to behave in Vim and not the other editors.

TL;DR: Not a bug. But, a lacking feature.

@mnquintana I think, this issue can be closed now and perhaps reopened later as a feature request.

**Comment by chenglou** [comment]
fwiw Sublime Text has the same behavior (not jumping)
